### ISSUE TITLE
Extension Manager: Sort plugins/templates by name

### ISSUE THREAD

**Issue by mightymt** [splitbrain/dokuwiki#1018]
Currently *Installed Plugins* (and also *Installed Templates*) in the Extension Manager are sorted by the name of their folder. However for quite a few plugins the actual name – as set in the *plugin.info.txt* – is different from the folder name and so the list is **not alphabetical**. 
I personally find this quite annoying sometimes if I want to quickly disable or enable a plugin. 

In my opinion they should be **sorted by the actual plugin (or template) name**.

Here's a screenshot to illustrate the issue. I have marked plugins which have differing names and cause the list to be jumbled up.

![extension_manager](https://cloud.githubusercontent.com/assets/4930561/5964590/b35fafbe-a7f5-11e4-9356-60e970cc00a8.png)

**Comment by kalenpw** [comment]
Should this be closed? Looks like it was done in https://github.com/splitbrain/dokuwiki/pull/2893

**Comment by phy25** [comment]
#2893 resolves unsorted plugin list by sorting it by plugin basename, which is exactly what is mentioned in this issue: "sorted by the name of their folder". And the author wants sorting by plugin title.

Probably this becomes a won't fix? Or this can be fulfilled by adding an option (parameter) on the interface?

**Comment by kalenpw** [comment]
I agree it should be sorted. In my opinion the best option would be when sorted by directory name display the name of the directory. When sorted by plugin name display the name of the plugin.

By adding a parameter are you referring to a query string for something like `sort=[plugin_name|directory-name]` and just choose the display based on that? With no sort given defaulting to plugin name?
